examples/app-mssql/main.py

```diff
@@ -0,0 +1,33 @@
+import pyodbc
+import sys
+import time
+
+# waiting for mssql db server to fully spawn
+time.sleep(3)
+
+print("Creating a mssql connection")
+conn = pyodbc.connect('DRIVER={ODBC Driver 17 for SQL Server};SERVER=mssql_server;DATABASE=tempdb;UID=SA;PWD=bad_P4ssword;')
```

Comment:
Instead of hardcoding credentials - (db, username, password, host) here please use environment variables. 

Something like this:
```suggestion
connection_str = 'DRIVER={ODBC Driver 17 for SQL Server};SERVER={server};DATABASE={database};UID={username};PWD={password};'.format(
    server=os.environ['DB_HOST'],
    database=os.environ['DB_NAME'],
    username=os.envion['DB_USERNAME'],
    password=os.environ['DB_PASSWORD']
)
conn = pyodbc.connect(connection_str)
```
You can then show the example of usage in the README.